tendrl/alerting/manager/__init__.py

```diff
@@ -0,0 +1,74 @@
+import logging
+import multiprocessing
+import signal
+from tendrl.alerting.central_store import AlertingEtcdPersister
+from tendrl.alerting.exceptions import AlertingError
+from tendrl.alerting.notification.exceptions import NotificationDispatchError
+from tendrl.alerting.notification import NotificationPluginManager
+from tendrl.alerting.watcher import AlertsWatchManager
+from tendrl.alerting.handlers import AlertHandlerManager
+
+
+LOG = logging.getLogger(__name__)
+
+
+class TendrlAlertingManager(object):
+    def __init__(self):
+        try:
+            tendrl_ns.alert_queue = multiprocessing.Queue()
+            tendrl_ns.notification_queue = multiprocessing.Queue()
+            self.notification_plugin_manager = NotificationPluginManager()
+            self.watch_manager = AlertsWatchManager()
+            self.alert_handler_manager = AlertHandlerManager()
+        except (AlertingError) as ex:
+            raise ex
+
+    def start(self):
+        try:
+            self.watch_manager.start()
+            self.notification_plugin_manager.start()
+            self.alert_handler_manager.start()
+        except (
+            AlertingError,
+            NotificationDispatchError,
+            Exception
+        ) as ex:
+            LOG.error(
+                'Exception %s' % str(ex),
+                exc_info=True
+            )
+            raise ex
+
+    def stop(self):
+        try:
+            tendrl_ns.alert_queue.close()
+            tendrl_ns.notification_queue.close()
+            self.notification_plugin_manager.stop()
+            self.alert_handler_manager.stop()
+            self.watch_manager.stop()
+            os.system("ps -C tendrl-alerting -o pid=|xargs kill -9")
+        except Exception as ex:
+            LOG.error(
+                'Exception %s' % str(ex),
+                exc_info=True
+            )
+            raise ex
+
+
+def main():
+    tendrl_ns.central_store_thread = AlertingEtcdPersister()
+    tendrl_ns.definitions.save()
+    tendrl_ns.config.save()
+
+    tendrl_alerting_manager = TendrlAlertingManager()
+
+    def terminate(sig, frame):
```

Comment:
@shtripat we won't be exposing any apis from alerting module anymore. Since alerting is treated as part of tendrl-core stack itself, the apis that were previously exposed by alerting module will now be exposed by tendrl/api